R/net.mod.infection.R

```diff
@@ -300,9 +312,11 @@ infection.2g.net <- function(dat, at) {
 #'
 discord_edgelist <- function(dat, at) {
 
-  status <- dat$attr$status
+  status <- get_attr(dat, "status")
+  active <- get_attr(dat, "active")
+  tgl <- get_control(dat, "tergmLite")
 
```

Comment:
Use the same name when you pull the object (i.e., tergmLite instead of tgl)

Reply:
Updated and fixed.